common/src/main/scala/models/VidispineProject.scala

```diff
@@ -0,0 +1,40 @@
+package models
+
+import java.time.ZonedDateTime
+import scala.util.Try
+import scala.xml.Node
+import java.time.format.DateTimeFormatter
+
+case class VidispineProject (vsid:String, name:Option[String], status:Option[String], commissionCollection:Option[String], deepArchive:Boolean, deletable:Boolean, sensitive:Boolean, created:Option[ZonedDateTime], modified:Option[ZonedDateTime])
+
+object VidispineProject {
+  def extractMetadataValue(metaNode:Node, fieldName:String):Option[String] = {
+    val applicableFieldNodes = (metaNode \ "timespan" \ "field").filter(fieldNode=>(fieldNode\"name").text==fieldName)
+    applicableFieldNodes.headOption.map(fieldNode=>(fieldNode\"value").text)
+  }
+
+  def checkField(field:Option[String], check:Option[String]): Boolean = {
```

Comment:
can simplify this method to `def checkField(field:Option[String], check:Option[String]): Boolean = field==check`

Reply:
Done.